gov/agendas/deployments.go

```diff
@@ -101,26 +102,31 @@ func (db *AgendaDB) loadAgenda(agendaID string) (*AgendaTagged, error) {
 // agendasForVoteVersion fetches the agendas using the vote versions provided.
 func agendasForVoteVersion(ver uint32, client DeploymentSource) (agendas []AgendaTagged) {
 	voteInfo, err := client.GetVoteInfo(ver)
-	if err == nil {
-		for i := range voteInfo.Agendas {
-			v := &voteInfo.Agendas[i]
-			agendas = append(agendas, AgendaTagged{
-				ID:             v.ID,
-				Description:    v.Description,
-				Mask:           v.Mask,
-				StartTime:      v.StartTime,
-				ExpireTime:     v.ExpireTime,
-				Status:         v.Status,
-				QuorumProgress: v.QuorumProgress,
-				Choices:        v.Choices,
-				VoteVersion:    voteInfo.VoteVersion,
-			})
-		}
+	if err != nil {
+		return
```

Comment:
Ah, crud, I think we should log err to help debug network connectivity.  Can you make the logging addition?

Reply:
I had an error being logged there initially but because it was always logged when vote version scanning took place I thought doing away with it would be better.

Reply:
The error made buck54321 think something was wrong https://github.com/decred/dcrdata/pull/1016/files/270b8a8bfa4fb616be3ef8e14d2631e1d7158be0#diff-4fe4b07854ee46957bec741a6797b39d

Reply:
The link above seems broken, here is the screenshot though.
<img width="858" alt="screenshot 2019-02-28 at 21 16 20" src="https://user-images.githubusercontent.com/22055953/53588535-2f8aae00-3b9e-11e9-89a9-ce2a7c9053ff.png">

Reply:
gotcha, I was thinking of the http.Client.  This is OK for the RPC client.